app_test.go

```diff
@@ -182,9 +184,11 @@ func ExampleApp_Run_commandHelp() {
 }
 
 func ExampleApp_Run_noAction() {
-	app := App{}
+	output := new(bytes.Buffer)
+	app := App{ErrWriter: output}
 	app.Name = "greet"
 	_ = app.Run([]string{"greet"})
+	fmt.Println(output.String())
```

Comment:
Nit: This looks like debug code—can we remove this line?

Reply:
It's not debug code and this is a testable example, i.e. after removing this line the testable example fails as this line prints to stdout what originally goes to ErrWriter :+1:

Reply:
Got it—the example requires we print to stdout, but we're now printing to stderr by default.

There's no need for a buffer to do this:

```go
app := App{ErrWriter: os.Stdout}
```

This should print to stdout like before without the buffer or the call to `fmt.Println`.

Reply:
Done